sdk/eventhub/event-hubs/src/connectionContext.ts

```diff
@@ -69,9 +69,9 @@ export namespace ConnectionContext {
    * @property userAgent The user agent string for the EventHubs client.
    * See guideline at https://github.com/Azure/azure-sdk/blob/master/docs/design/Telemetry.mdk
    */
-  const userAgent: string = `azsdk-js-azureeventhubs/${packageJsonInfo.version} (NODE-VERSION ${
-    process.version
-  }; ${os.type()} ${os.release()})`;
+  const userAgent: string = `azsdk-js-azureeventhubs/${
+    packageJsonInfo.version
+  } (NODE-VERSION ${getRuntimeInfo()})`;
```

Comment:
So... we are going to say "NODE-VERSION" even in browser?

Reply:
I had a reason but now I think it's not a good one, I'll create a new PR to change this to BROWSER_VERSION.

Reply:
Also, looks like the entire `runtime info` thing is repeated in event hubs. Can there be a way where we avoid this dance in event hubs? How about we send just `azsdk-js-azureeventhubs/version` in user agent string and then have `core-amqp` figure out node version and runtime info and suffix it?

Reply:
I'm not sure we can safely do that because someone using core-amqp directly could be setting their own user-agent string that we'd be adding a suffix to unexpectedly. We have examples in our readme for core-amqp on how to set the user agent string.